helpick-service/src/main/java/com/infoshare/database/FileDb.java

```diff
@@ -77,6 +78,42 @@ public void saveVolunteer(Volunteer volunteer) {
 
     }
 
+    @Override
+    public void saveVolunteerWithUuid(Volunteer volunteer){
+        List<Volunteer> allVolunteers = getVolunteers();
+        boolean isNewVolunteer = true;
+        for (int i = 0; i < allVolunteers.size(); i++) {
+            Volunteer volunteerFromDb = allVolunteers.get(i);
+            if (volunteerFromDb.getUuid().equals(volunteer.getUuid())){
+                isNewVolunteer = false;
+                allVolunteers.set(i, volunteer);
+                break;
+            }
+            else if (volunteerFromDb.getEmail().equalsIgnoreCase(volunteer.getEmail()) ) {
+                return;
+            }
+        }
+        if(isNewVolunteer){
+            allVolunteers.add(volunteer);
+        }
+
+        try{
+            try (FileWriter writer = new FileWriter(VOLUNTEER_DB_FILE_NAME, false)) {
+                for (Volunteer v : allVolunteers) {
+                    writer.write(v.getName() + "," +
+                            v.getLocation() + "," +
+                            v.getEmail() + "," +
+                            v.getPhone() + "," +
+                            v.getTypeOfHelp() + "," +
+                            v.isAvailable() + "," +
+                            volunteer.getUuid() + "\n");
```

Comment:
Tutaj jest ten sam blad co byl w saveVolunteer, czyli powinno byc
```suggestion
                            v.getUuid() + "\n");
```

Reply:
poprawione :+1: